stablehlo/reference/Ops.cpp

```diff
@@ -81,5 +94,17 @@ Tensor eval(TanhOp op, const Tensor &operand) {
   return result;
 }
 
+Tensor eval(TransposeOp op, const Tensor &operand) {
+  Tensor result(op.getType());
+  SmallVector<int64_t> permutation =
+      to_vector(op.permutation().getValues<int64_t>());
```

Comment:
Can we avoid `to_vector` and just pass `getValues` directly? (By appropriately changing the signature of `permute`).

Reply:
Sure. The signature looks verbose, but, being a static function, does not hurt.